src/prefect/agent.py

```diff
@@ -44,10 +47,11 @@
 from prefect.states import Crashed, Pending, StateType, exception_to_failed_state
 
 
+@deprecated_class(
+    start_date="Mar 2024",
+    help="Use a worker instead. Refer to the upgrade guide for more information: https://docs.prefect.io/latest/guides/upgrade-guide-agents-to-workers/.",
+)
 class PrefectAgent:
-    @experimental_parameter(
-        "work_pool_name", group="work_pools", when=lambda y: y is not None
-    )
```

Comment:
is this just something that should have been removed a while ago? it's not associated with the deprecation, right?

Reply:
Yeah, it should've been removed a while ago. It hasn't been emitting because the work pool feature defaults to enabled.